### DISCUSSION TITLE
충돌처리 OnCollisionEnter은 어떻게 구현하나요?

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#273] [5. Q&A]
OnTriggerEnter은 잘 작동하는거 같은데, OnCollisionEnter은 구현이 안되네요.

혹시 어떻게 해결할 수 있을까요??

**Comment by x4cf** [comment]
OnCollisionEnter 는 사용불가능합니다. OnTriggerEnter 를 이용해서 구현을 해보는건 어떨까요?
OnCollisionEnter 로 구현하는 기능중에 OnTriggerEnter 로 구현 불가능한건 없답니다 :)

**Reply by x4cf** [reply]
와우! 사용가능하다고 하네요!
저도 하나 알아가요 XD

**Comment by hyeonjin-jo** [comment]
안녕하세요.

OnCollisionEnter 의 경우 유니티와 동일한 방식으로 사용이 가능합니다.

아래 링크를 참고해 주시고 다른 세팅에 문제가 없는지 확인 부탁드립니다.
https://docs.unity3d.com/kr/530/ScriptReference/Collider.OnCollisionEnter.html

**Reply by hipsc159** [reply]
죄송한데 부가 설명 가능하실까요..? 
OnCollisionEnter 함수가 아예 활성화 안되는거 같은데 어떤 방식으로 해결하는 것이죠?

**Reply by hyeonjin-jo** [reply]
우선 C#환경에서 OnCollisionEnter가 동작하게 만들고 ts로 변환하는 것을 추천드립니다
unity에서 OnCollisionEnter를 사용하기위해 리지드바디 세팅 등등이 잘되어있는지 확인 부탁드립니다
